mne_features/univariate.py

```diff
@@ -609,7 +613,7 @@ def compute_pow_freq_bands(sfreq, data, freq_bands=np.array([0.5, 4., 8., 13.,
     n_channels = data.shape[0]
     fb = _freq_bands_helper(sfreq, freq_bands)
     n_freq_bands = fb.shape[0]
-    psd, freqs = power_spectrum(sfreq, data, return_db=False)
+    psd, freqs = power_spectrum(sfreq, data, method=psd_method)
```

Comment:
you don't expose any of the parameters of Welch or mutlitaper? like the window length and overlap for welch

Reply:
I hesitated to expose them but you're right. I should have !